consensus/replay_test.go

```diff
@@ -149,17 +150,10 @@ func runReplayTest(t *testing.T, cs *ConsensusState, walFile string, newBlockCh
 	// Assuming the consensus state is running, replay of any WAL, including the empty one,
 	// should eventually be followed by a new block, or else something is wrong
 	waitForBlock(newBlockCh, thisCase, i)
-	cs.evsw.Stop()
 	cs.Stop()
-LOOP:
```

Comment:
because everything is synchronous. so cs would block on firing on newBlockCh while cs.Wait is waiting for it to exit -> deadlock